data/uk-cs-dictionary.json

```diff
@@ -1867,7 +1954,8 @@
 				"rec3Xdb5QAjNFr7ob",
 				"recmYthRZ57qOfBVU"
 			],
-			"hidden": true
+			"hidden": true,
+			"metacategories": []
```

Comment:
Nebylo by lepší `"metacategoriew": null`, když je to nullable proměnná?

Reply:
mě se prázdné pole líbí, zjednodušuje to myslím implementaci (pokud teda není rozdíl - nikdy nenastaveno a prázdný seznam) :)